src/pages/Success.tsx

```diff
@@ -69,20 +69,17 @@ const Success = () => {
   // specify order of animations, will be animated one after the other
   useChain([emojiAnimRef, appearAnimRef, pointsAnimRef, panelAnimRef])
 
-  const updateScores = () => {
+  const goToTriviaScreen = () => {
     // set the points here, as we wanted to animate the difference
     addPoints(pointDifference)
-    // reset point difference for next round
+    // reset point difference for next round. we only do that when we go to the trivia screen
```

Comment:
Important to keep around the points won in the normal question, such that they can be doubled in the bonus round (or lost!)